src/pages/wallet-details/wallet-details.html

```diff
@@ -33,22 +33,13 @@
 
         <div (longPress)="toggleBalance()">
           <div (tap)="updateAll(true)" *ngIf="!wallet.balanceHidden && !wallet.scanning && !walletNotRegistered && (wallet.cachedStatus || wallet.lastKnownBalance)">
-            <div *ngIf="wallet.coin !== 'xrp'; else xrpBalance">
+            <div (click)="wallet.coin === 'xrp' ? openBalanceDetails() : ''">
```

Comment:
I think we can't have `(tap)` and `(click)` into the same scope. It could have a weird behavior. Maybe the function should be relocated as an (i)nfo icon below.

Reply:
added info icon for click event. Thanks!